modules/swagger-codegen/src/main/resources/META-INF/services/io.swagger.codegen.CodegenConfig

```diff
@@ -7,7 +7,9 @@ io.swagger.codegen.languages.FlaskConnexionCodegen
 io.swagger.codegen.languages.GoClientCodegen
 io.swagger.codegen.languages.JavaClientCodegen
 io.swagger.codegen.languages.JavascriptClientCodegen
-io.swagger.codegen.languages.JaxRSServerCodegen
+io.swagger.codegen.languages.JavaClientCodegen
+io.swagger.codegen.languages.JavaJerseyCodegen
+io.swagger.codegen.languages.JavaCXFCodegen
```

Comment:
It's ok for me. Since the PR is in your repo, who has to do the change ? Me
or you ?



-- 

*Maëlig NANTEL*
Mobile : +33 (0)6 69 47 39 00
Mail : *maelig.nantel@gmail.com <maelig.nantel@gmail.com>*